CondFormats/L1TObjects/interface/CaloParams.h

```diff
@@ -128,9 +128,13 @@ namespace l1t {
       // veto region is seed tower +/- <=egIsoVetoNrTowersPhi
       unsigned isoVetoNrTowersPhi_;
 
+      // turn on/off EG ID cuts
```

Comment:
Made a new PR #16324
Got rid of the newly added data members (bools egBypassEGVetos_ and jetBypassPUS_ ) in CondFormats, which are currently always used, set and configured, as false. 

This avoids any need for a new GlobalTag.